.flake8

```diff
@@ -0,0 +1,6 @@
+[flake8]
+extend-ignore = B902,C816,D100,D101,D102,D103,D104,D105,D106,D107,D203,D212,D404,I202
```

Comment:
@nahueespinosa I'm curious about B902. It does seem to indicate a bug.

Reply:
This was the configuration used by `ament_flake8`.
https://github.com/ament/ament_lint/blob/rolling/ament_flake8/ament_flake8/configuration/ament_flake8.ini

I kept only the ones that currently fail in https://github.com/Ekumen-OS/beluga/pull/237/commits/01552b9b253d9b1c4b6aa3025f46b4da105b8f55.

Reply:
It's the [ament_flake8](https://github.com/ament/ament_lint/blob/f10eecd28cdc05746c1f8c0f45984db6052cfd40/ament_flake8/ament_flake8/configuration/ament_flake8.ini#L2) style.

IMO, we could use standard flake8 without any ignore.
(some may be required to be compatible with black)